README.md

```diff
@@ -75,14 +75,38 @@ FROM alpine:edge
 RUN apk --no-cache add bash ca-certificates
 
 WORKDIR /usr/local/bin
-RUN wget https://raw.githubusercontent.com/vishnubob/wait-for-it/db049716e42767d39961e95dd9696103dca813f1/wait-for-it.sh && \
+RUN wget https://raw.githubusercontent.com/vishnubob/wait-for-it/ed77b63706ea721766a62ff22d3a251d8b4a6a30/wait-for-it.sh && \
     chmod +x wait-for-it.sh
 
 CMD ["cat", "/app/test_files/zzz.txt"]
 ```
 
 wait-for-it requires bash, so bash is added, then the  wait-for-it script is downloaded, and made executable.
 
+### Ignoring files
+
+The docker compose `command` can be used to pass additional options to Unison. A good usage for this
+is to ignore files.  For example if you want to ignore all log files you can use this:
+
+```yaml
+...
+services:
+  sync:
+    image: concordconsortium/docker-volume-sync
+    command: ["-ignore", "Name *.log"]
+    volumes:
+      - sync-volume:/data
+      - .:/host_data
+...
+```
+More info about the syntax for ignoring files is here:
+https://www.cis.upenn.edu/~bcpierce/unison/download/releases/stable/unison-manual.html#ignore
+
+If you are wanting to ignoring a folder on your host system, you might be able to optimize performance further by adding an additional
```

Comment:
small suggestion: "wanting to ignoring" → "want to ignore"

Is it  possible to ignore multiple paths, files, or patterns using this method, if so, how would that be done?  Are the images default ignores ignored?